server/graphql_schemas/event/schema.py

```diff
@@ -113,8 +114,14 @@ def mutate_and_get_payload(cls, root, info, **input):
         except ValueError as e:
             raise GraphQLError("An Error occurred. \n{}".format(e))
 
+        slack_token = False
+        if (user_profile.slack_token):
+            slack_token = Truegst
```

Comment:
Nice work @njeri-ngigi. It seems there is a typo here. Kindly help fix that. Thanks
```suggestion
            slack_token = True
```

Reply:
thanks for catching that